src/components/text-editor/__internal__/toolbar/toolbar-button/toolbar-button.style.ts

```diff
@@ -16,11 +18,20 @@ const StyledToolbarButton = styled.button.attrs({ type: "button" })<{
     width: auto;
   }
 
-  ${({ isActive }) => css`
+  ${({ isActive, theme }) => css`
     :focus,
     :active {
-      outline: 2px solid var(--colorsSemanticFocus500);
-      outline-offset: -2px;
+      ${theme.focusRedesignOptOut &&
```

Comment:
**suggestion** (non-blocking): as commented elsewhere, this could be a ternary, and also you've not extracted the `oldFocusStyles` to a const as you have in most other components